File: contains_duplicate_1/main.go
package main

import "fmt"

func containsDuplicate(nums []int) bool {
	if len(nums) <= 1 {
		return false
	}
	hashmap := make(map[int]int)
	for _, num := range nums {
		if _, ok := hashmap[num]; ok {
			return true
		} else {
			hashmap[num] = 1
		}
	}
	return false
}

func main() {
	fmt.Println(containsDuplicate([]int{1, 2, 3, 1}))
	fmt.Println(containsDuplicate([]int{1, 2, 3, 4}))
}

File: contains_duplicate_1/main.py
from typing import List

def containsDuplicate(nums: List[int]) -> bool:
    if len(nums) ==1:
        return False
    hashmap = {}
    for num in nums:
        if num in hashmap:
            return True
        hashmap[num] =1
    return False

if __name__ == "__main__":
    assert containsDuplicate([1,2,3,1]) , "Wrong answer"